### PR TITLE
fix(cx 10437) : Outdial Call: Screen Readers: Combobox expand/collapse button focused separately

### PR DESCRIPTION
**Pull request by neelabhnagaich**
## Description
CX-10437

## Screenshots:
**Before** (If applicable):

**After**:

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by akshatatray — CHANGES_REQUESTED** [review]
Update patch version

File: web-components/src/components/combobox/ComboBox.ts [hunk 1]
```diff
@@ -1350,10 +1350,10 @@ export namespace ComboBox {
         <button
           type="button"
           class="md-combobox-button arrow-down"
-          aria-label=${this.arrowAriaLabel}
           aria-expanded=${this.expanded}
           aria-controls="md-combobox-listbox"
           tabindex="-1"
+          aria-hidden="true"
```
Comment (vinitkumargoel) [inline web-components/src/components/combobox/ComboBox.ts:1356]: Take a property dont use hard code value

File: web-components/src/components/combobox/ComboBox.ts [hunk 2]
```diff
@@ -1368,9 +1368,9 @@ export namespace ComboBox {
         <button
           type="button"
           class="md-combobox-button"
-          aria-label=${this.arrowAriaLabel}
           aria-controls="md-combobox-listbox"
           tabindex="-1"
+          aria-hidden="true"
```
Comment (vinitkumargoel) [inline web-components/src/components/combobox/ComboBox.ts:1373]: Take a property dont use hard code value

Comment (neelabhnagaich) [reply web-components/src/components/combobox/ComboBox.ts]: done

File: web-components/src/components/combobox/ComboBox.ts (hunk 1 above)
Comment (neelabhnagaich) [reply web-components/src/components/combobox/ComboBox.ts]: done

File: web-components/src/components/combobox/ComboBox.ts [hunk 3]
```diff
@@ -82,6 +82,7 @@ export namespace ComboBox {
     @property({ type: Boolean, attribute: "show-custom-error", reflect: true }) showCustomError = false;
     @property({ type: Boolean, attribute: "show-loader", reflect: true }) showLoader = false;
     @property({ type: Boolean, attribute: "show-selected-count", reflect: true }) showSelectedCount = false;
+    @property({ type: String, attribute: "popup-indicator-aria-hidden" }) popupIndicatorAriaHidden = "true";
```
Comment (merajcal) [inline web-components/src/components/combobox/ComboBox.ts:85]: rename the prop from popup-indicator-aria-hidden to popup-chevron-aria-hidden

**Review by merajcal — CHANGES_REQUESTED** [review]

File: web-components/src/components/combobox/ComboBox.ts (hunk 3 above)
Comment (neelabhnagaich) [reply web-components/src/components/combobox/ComboBox.ts]: done

**Review by merajcal — APPROVED** [review]

**Review by vinitkumargoel — APPROVED** [review]

**Pull request merged**